sdk/mediaservices/Azure.Media.Analytics.Edge/samples/LiveVideoAnalyticsSample.cs

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.Core.TestFramework;
+using Azure.Media.Analytics.Edge.Models;
+using Microsoft.Azure.Devices;
+using NUnit.Framework;
+using Newtonsoft.Json.Linq;
+using System.Threading.Tasks;
+using System.IO;
+using System.Text;
+using Azure.Core;
+using System.Linq;
+
+namespace Azure.Media.Analytics.Tests.Samples
+{
+    public class LiveVideoAnalyticsSample
```

Comment:
Why do you have [this](https://github.com/Azure/azure-sdk-for-net/tree/df5f714e8567bed66db1c4478bc74e8b542d3786/sdk/mediaservices/Azure.Media.Analytics.Edge/tests/Samples) Samples folder?

I would suggest to follow the sample structure as [this](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples) SDK, meaning create md files inside[ samples folder](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples), add the samples inside tests folder(so that we can run the samples as a part of tests) and add the samples link at the end of the md files. For reference check [this ](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample1_RecognizeFormContent.md)md file.

Reply:
This is a work around for right now since we don't want to run live tests. We can update with the next release to include live tests if that is alright

Reply:
If we are not doing anything [here](https://github.com/Azure/azure-sdk-for-net/tree/df5f714e8567bed66db1c4478bc74e8b542d3786/sdk/mediaservices/Azure.Media.Analytics.Edge/tests/Samples), can we remove it?

Reply:
Can we create md files and explain what exactly we are doing in the samples? and add samples link at the end of the files?